.gitignore

```diff
@@ -56,6 +56,11 @@ cmake-build-*/
 .kdev?/
 *.kdev?
 
+##########
+# VsCode #
+##########
+.vscode
+
```

Comment:
This change is ok here, but would be better in a separate PR since it isn't related to the main changes.

Reply:
I can revert it and propose a new small PR for it, no problem for me.